__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const app = require('../app')
+const db = require('../db/connection')
+const request = require('supertest')
+const data = require(`../db/data/test-data/index`)
+const seed = require('../db/seeds/seed')
+
+
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll((() => db.end()))
+
+describe('General app test', () => {
+    test('GET 404 - responds with 404 and `Endpoint not found` if requested endpoint has not been defined', () => {
+        return request(app)
+            .get('/api/no_api_here')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Endpoint not found')
+            })
+    })
+})
+describe('GET /api/topics', () => {
+    test('GET 200 a valid request should return a status code 200', () => {
```

Comment:
This test could absolutely be combined with the test below. Remember we're integration testing not unit testing. There will be some cases where one test is totally enough to prove it works.